<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamUi</class>
 <widget class="QMainWindow" name="ParamUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Normal_GG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>begin_step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditbegin_step"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>end_step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditend_step"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>int_width</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditint_width"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Nz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditNz">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Ny</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditNy">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditNx"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_25">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>dt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditdt">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>dx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditDx">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>phi_incre_limit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditphi_incre_limit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>file_output_step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditfile_output_step"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>screen_output_step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditscreen_output_step"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>dataFile_output_step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditdataFile_output_step"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>grain_number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditgrain_number"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>generate_step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditgenerate_step"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>box_position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_box_positionX"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_box_positionY"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_box_positionZ"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>box_size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditbox_sizeX"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditbox_sizeY"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditbox_sizeZ"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_23">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Xi_abc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditXi_abc"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>int_grad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxint_grad">
        <item>
         <property name="text">
          <string>Steinbach_1996</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Steinbach_1999</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Steinbach_G2009</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Int_GCustom</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>int_pot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxint_pot">
        <item>
         <property name="text">
          <string>Nestler_Well</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nestler_Obstacle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Steinbach_P2009</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Int_PCustom</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>is_write_datafile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxis_write_datafile">
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>is_init_byDatafile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxis_init_byDatafile">
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>isPhiOutput</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxisPhiOutput">
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
      <item>
       <widget class="QPushButton" name="pushButtonRun">
        <property name="text">
         <string>run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditExeDir"/>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
